packages/@aws-cdk/aws-lambda-nodejs/lib/function.ts

```diff
@@ -155,7 +155,7 @@ function findEntry(id: string, entry?: string): string {
     return jsHandlerFile;
   }
 
-  throw new Error('Cannot find entry file.');
+  throw new Error(`Cannot find handler file ${jsHandlerFile}`);
```

Comment:
This maybe? (to show that it also searches for ts files)
```suggestion
  throw new Error(`Cannot find handler file ${tsHandlerFile} or ${jsHandlerFile}`);
```